rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/dependencyreport/CreateDependencyReportRuleProvider.java

```diff
@@ -150,7 +149,7 @@ private void createGlobalReport(GraphContext context, WindupConfigurationModel c
             addAll(context, reportModel, traversal, sha1ToGroup);
         }
 
-        reportModel.setDisplayInGlobalApplicationIndex(true);
+        reportModel.setDisplayInGlobalApplicationIndex(Boolean.TRUE);
```

Comment:
what's the difference ? the autoboxing would directly convert to Boolean, right ?

Reply:
lol, I noticed that and wondered the same thing.

Reply:
....hhhmmm....other than this change, i don't know why this class has been changed since it's not involved in implementing this feature and i'm quite sure i didn't change it :thinking: 

I figured out what happened: it was part of the initial commit from Janine [CreateDependencyReportRuleProvider](https://github.com/jeichler/windup/blob/dependencies-graph/rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/dependencyreport/CreateDependencyReportRuleProvider.java#L152) so i didn't see it so thanks for highlighting this.
We're not supposed to avoid the autoboxing so this kind of changes are not expected to be done.